### PR TITLE
V2 Improvements based on 2nd review

### PR DESCRIPTION
**Pull request by pkuttappan**
This PR includes the following changes:

- Modified subItem selector behaviour from selecting items globally to selecting items under subItem selector
- Added onload functionality to subItem (adds opacity to excluded items on loading subItem config)
- Modified the scroll behavior -  added scroll to the parent box instead of individual metadata/exclude container
- Made the toast message position sticky - added toast on Save file and subItem save 
- Added a new attribute hasChanges to keep track of file changes and added an alert if file is closed before saving the changes.

Whats left:
- Adding the flex layout

### CODE REVIEW

File: chrome_extension/content-script.js [hunk 1]
```diff
@@ -86,17 +93,22 @@ function applyStylesToElements(newItem, oldItem = null) {
   }
 }
 
-function removePreviouslyExcludedStyles() {
-  document.querySelectorAll('.web-scraper-helper-exclude').forEach(e => {
-    if (e && e.classList) {
-      e.classList.remove('web-scraper-helper-exclude');
-    }
-  });
+function removePreviouslyExcludedStyles(parentSelector = null) {
```
Comment (jdevost) [inline chrome_extension/content-script.js:96]: In this case, we want to remove the class in the whole page, we don't need to consider the `parentSelector`. You can revert this function.

**Review by jdevost — COMMENTED** [review]
When removing the class `web-scraper-helper-exclude` from the page, we don't need to consider the parent. We want to do it globally when removing.

File: chrome_extension/content-script.js (hunk 1 above)
Comment (pkuttappan) [reply chrome_extension/content-script.js:96]: This function is called when a subItem config file is closed too. Hence I used the parent selector here.

Comment (jdevost) [reply chrome_extension/content-script.js:96]: 🤔 Does it mean the excluded items from subitems aren't hidden if we aren't in the tab for subitems?
I think we should hide them generally. We can discuss it.

**Review by jdevost — APPROVED** [review]
Looks good.

**Pull request merged**